pkg/template/core/starlark_struct.go

```diff
@@ -40,3 +45,50 @@ func (s *StarlarkStruct) AttrNames() []string {
 	})
 	return keys
 }
+
+func (s *StarlarkStruct) Get(key starlark.Value) (val starlark.Value, found bool, err error) {
+	if attr, ok := NewStarlarkValue(key).AsGoValue().(string); ok {
+		val, err = s.Attr(attr)
+		return val, val != nil, err
+	} else {
+		return nil, false, fmt.Errorf("expected key `%s` to be a string but is a %T", key, key)
+	}
+}
+
+func (s *StarlarkStruct) Iterate() starlark.Iterator {
+	return &StarlarkStructIterator{
```

Comment:
one liner?

Reply:
eh. When I tried it, it was _less_ readable.